docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -175,15 +173,32 @@ The output should be
 [source,bash,subs="attributes+"]
 ----
 curl -i -H "$AUTH_HEADER" \
-  -F url="$(mc share download --json "$\{CLUSTER_ID\}/$\{CLUSTER_ID\}-bootstrap-ignition/rhcos-{ocp-minor-version}.raw" | jq -r .url)" \
+  -F url="$(mc share download --json "$\{CLUSTER_ID\}/$\{CLUSTER_ID\}-bootstrap-ignition/rhcos-{ocp-minor-version}.qcow2" | jq -r .url)" \
   -F name='RHCOS {ocp-minor-version}' \
   -F zones="${REGION}1" \
   -F slug=rhcos-{ocp-minor-version} \
-  -F source_format=raw \
+  -F source_format=qcow2 \
   -F user_data_handling=pass-through \
   https://api.cloudscale.ch/v1/custom-images/import
 ----
 
+. Wait for custom image import to complete
++
+[source,bash,subs="attributes+"]
+----
+while [ "$(curl -s -H $AUTH_HEADER https://api.cloudscale.ch/v1/custom-images/import | jq -r '.[].status')" != "success" ]; do
```

Comment:
This won't work if there's already another image uploaded in the same cloudscale.ch organization since this command lists all upload statuses and compares a potential multi-line value with the string "success".

You should probably get the image uuid of the current import in the previous step with `image_uuid=$(curl ... | jq -r '.uuid')` and use it here to only check the upload status for the current import with `https://api.cloudscale.ch/v1/custom-images/import/${image_uuid}`.

Reply:
Would it make more sense to just put the image deletion step at the end of the how-to? Like that you also wouldn't have to wait for the image import to finish (takes like 2 min).

Reply:
Yeah, assuming we generally will spend >=2 minutes until we actually need the image, I'm fine with just cleaning up at the end of the how-to (we can probably actually completely empty and remove the bootstrap bucket at that point).

Reply:
I tested the command in a project where there were already custom images imported at an earlier stage, so it looks like they don't keep those import status  info indefinitely. IIRC I read that failed imports disappear after one week, so it's probably the same for successful imports.